src/mrd/ugbio_mrd/mrd_utils.py

```diff
@@ -9,15 +9,20 @@
 from os.path import basename, dirname, splitext
 from os.path import join as pjoin
 
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import pyBigWig as bw  # noqa: N813
 import pysam
+import seaborn as sns
+from IPython.display import display
```

Comment:
I think it's best to remove that part, and add the "display" command in the notebook when we want to show a table.